help/md/fnCovarion.md

```diff
@@ -1,9 +1,56 @@
 ## name
 fnCovarion
 ## title
+The Covarion model rate matrix.
 ## description
+The `fnCovarion` function defines a covarion model rate matrix for character evolution.
+The resulting rate matrix incorporates rate heterogeniety where the characters/sites are allowed to move between rate categories with a switching rate.
+
 ## details
+The covarion model allows for variation in evolutionary rates across sites over time, accomodating shifts in character state evolution.
+
 ## authors
+Sebastian Hoehna and Lyndon Coghill
+
 ## see_also
+`fnJC` - Jukes-Cantor Model
+
+`fnF81` - Felsenstein 1981 Model
```

Comment:
I'm not sure the "See also" items will be properly converted to links during the website-building process if they are placed between ticks and followed by explanatory notes...